### ISSUE TITLE
Selection article: Duplicate entry

### ISSUE THREAD

**Issue by noll1** [aimeos/aimeos-typo3#80]
I installed aimeos in a fresh Typo3 v9.5.0 LTS environment and opened the "Demo selection article".
Without any changes I click save and get the following error:

Error SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1-attribute-7-4-2' for key 'unq_msproli_sid_dm_rid_tid_pid', /var/www/typo3/typo3conf/ext/aimeos/Resources/Libraries/aimeos/aimeos-core/lib/mwlib/src/MW/DB/Statement/PDO/Simple.php:92

My debugger tells me this SQL throws the exception:
```
INSERT INTO "mshop_product_list" (
	"parentid",
	"typeid",
	"domain",
	"refid",
	"start",
	"end",
	"config",
	"pos",
	"status",
	"mtime",
	"editor",
	"siteid",
	"ctime"
	)
VALUES (
	2,
	4,
	'attribute',
	'7',
	NULL,
	NULL,
	'[]',
	0,
	1,
	'2018-10-29 13:57:01',
	'admin',
	1,
	'2018-10-29 13:57:01'
	)
```

Also it is not possible to create another selection article with more than one variant.

**Comment by aimeos** [comment]
This is a known issue already fixed in the 18.10 branch of the Aimeos extension. If you've installed TYPO3 und Aimeos using composer, you can simply run `composer update`. If not, a new package will be released in TER tomorrow.

**Comment by noll1** [comment]
Ok, thank you for the information.
